x-pack/plugins/viz_editor/server/query/to_es_query.test.ts

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { toEsQuery } from './to_es_query';
+
+describe('viz-editor/query/to_es_query', () => {
+  test('performs a limit without aggregations', () => {
+    expect(
+      toEsQuery({ index: 'a', select: [{ operation: 'col', argument: 'datacenter' }], size: 20 })
+    ).toMatchObject({
+      size: 20,
+    });
+  });
```

Comment:
I would have expected this to also limit the fields in the response, such as `fields: ['datacenter']`, why isn't it? In the example below on line 24, it does...?

Reply:
It does. This test isn't really for that scenario, so I don't want this to break if we change that behavior (we should test that in its own unit test). This one is only testing that size gets properly transferred to the es query.

Reply:
It does affect that. This unit test is only checking that size gets properly set in the generated es query. I wrote a lot of tests for migrations that were testing things like this, but also incidentally compared lots of other (unrelated) properties in the result. That meant that changes often involved fixing lots of tests which really shouldn't have cared about the change.